doc/source/design.rst

```diff
@@ -51,6 +51,54 @@ behavior of the module. These tests should be simple and short, testing
 a single function each, and completing as quickly as possible
 (ideally < 10 sec, but times up to a minute are acceptable if justified).
 
+Benchmarks
+==========
+
+Modules can be benchmarked in the following way. 
+
+.. code-block:: python
+    
+    # benchmarks/benchmark_example_Example.py
+    from benchmark import Benchmark
```

Comment:
Missing something like `from benchmarker import run_benchmarks`

Please check this to be sure this example file matches the implemented design. To make it simpler, we could just include a real benchmark with [the `.. literalinclude::` directive](https://www.sphinx-doc.org/en/1.5/markup/code.html#directive-literalinclude) and say "The following code is an example benchmark for the `freud.density.RDF` module," etc.